### DISCUSSION TITLE
feature request: more UI options (tab title color, window borders, and new tab animation)

### DISCUSSION THREAD

**Discussion by ravibrock** [ghostty-org/ghostty#3597] [Feature Requests, Ideas]
Hi! I just started using Ghostty, and really like it so far. Definitely planning on daily driving it for the foreseeable future. That said, there were a few small things I noticed and was hoping to request them (especially if they're quick to implement):

- It would be nice if there were an option to change the color of the tab title (specifically with `macos-titlebar-style = "tabs"`). I was hoping to make it more gray rather than the current bright white. See image 1 for reference.
- If possible, there would be a way to toggle the gray window border--I think it's a little cleaner without; this isn't a huge deal though obviously. I know that it either doesn't exist or is optionally disabled on iTerm. See image 2 for reference.
- Disabling new tab/tab opening animations (specifically with the aforementioned MacOS titlebar style) would be amazing. I find it really distracting when the tab bar jitters/slides upon opening a new tab.

Thank you!

Image 1:
![Screenshot 2024-12-26 at 15 36 31](https://github.com/user-attachments/assets/7682526d-1e47-41f1-a3f2-8d94a8731f5c)

Image 2:
![Screenshot 2024-12-26 at 15 36 16](https://github.com/user-attachments/assets/263b133c-c278-4a89-98d7-891f3a88dd40)

**Comment by nihil2501** [comment]
_Answer generated by ChatGPT_
> - Disabling new tab/tab opening animations (specifically with the aforementioned MacOS titlebar style) would be amazing. I find it really distracting when the tab bar jitters/slides upon opening a new tab.
### Reduce Motion Setting
- Go to **System Settings → Accessibility → Display**
- Enable **Reduce Motion**
- This setting reduces or disables animations across the system, including tab animations in many apps

**Note:** This might affect other animations on macOS, such as opening Mission Control or switching between workspaces.

**Comment by adonh** [comment]
adding 👍 for this feature, too. TY!
